third_party/nvfuser/python_tests/test_python_frontend.py

```diff
@@ -987,5 +989,36 @@ def fusion_func(fd: FusionDefinition):
         eager_out = torch.full([2, 2], 1.0) * 5.0
         self.assertEqual(eager_out, nvf_out[0]) 
 
+    def test_nextafter(self):
+        inputs = [
+            # torch.nextafter is only defined for float{32,64} tensor inputs
+            make_tensor(4, device="cuda", dtype=torch.float32),
+            make_tensor(4, device="cuda", dtype=torch.float64),
+        ]
+
+        def fusion_func(fd: FusionDefinition):
+            t0 = fd.from_pytorch(inputs[0])
+            t1 = fd.from_pytorch(inputs[1])
+
+            s0 = fd.define_constant(1.0, dtype=DataType.Float)
+            s1 = fd.define_constant(-1.0, dtype=DataType.Double)
+
+            t2 = fd.ops.add(t0, s0)  # float
+            t3 = fd.ops.add(t1, s1)  # double
+
+            for a, b in itertools.product(
+                [t0, t1],#, s0, s1],
```

Comment:
Tests with scalar args are currently commented out. I'm hitting a segfault (different from the `addcmul` one) in `Val::isConstScalar`. I'll investigate this and fix before this PR is ready to merge.